src/Components/Artwork/Details.tsx

```diff
@@ -89,16 +90,71 @@ export class Details extends React.Component<Props, null> {
     }
   }
 
-  render() {
+  saleInfoLine() {
+    const { artwork } = this.props
+    const { sale } = artwork
+    const inClosedAuction = sale && sale.is_auction && sale.is_closed
+
     return (
       <div>
-        {this.artistLine()}
-        {this.titleLine()}
-        {this.partnerLine()}
-        {this.props.showSaleLine && this.saleLine()}
+        <div>
+          {inClosedAuction ? "Bidding closed" : this.saleMessageOrBidInfo()}
+          {!inClosedAuction && this.auctionInfo()}
+        </div>
       </div>
     )
   }
+
+  saleMessageOrBidInfo() {
+    const { artwork } = this.props
+    const { sale } = artwork
+    const inRunningAuction = sale && sale.is_auction && !sale.is_closed
+
+    if (inRunningAuction) {
+      if (sale && sale.is_open) {
+        const sa = artwork.sale_artwork
+        const bids = sa && sa.counts && sa.counts.bidder_positions
+        if (bids && bids > 0) {
+          return sa.highest_bid.display
+        } else {
+          return sa.opening_bid.display
+        }
+      }
+    } else {
+      return artwork.sale_message
+    }
+  }
+
+  auctionInfo() {
+    const { artwork } = this.props
+    const { sale } = artwork
+
+    if (sale) {
+      return `(${sale.display_timely_at})`
```

Comment:
`display_timely_at` is the field in MP that I'll modify to more correctly express the data in the comp (some formatting, as well as handling 'register by' which will check if the current user is registered for that sale).

Reply:
Ah cool, yeah I believe I initially implemented `display_timely_at` a while back for Emission auctions home page